Cargo.toml

```diff
@@ -150,10 +150,10 @@ openvm-native-compiler = { git = "https://github.com/openvm-org/openvm.git", tag
 openvm-native-recursion = { git = "https://github.com/openvm-org/openvm.git", tag = "v1.1.2", default-features = false }
 
 # powdr
-powdr-openvm = { git = "https://github.com/powdr-labs/powdr.git", rev = "c2ef9e4", default-features = false }
-powdr-riscv-elf = { git = "https://github.com/powdr-labs/powdr.git", rev = "c2ef9e4", default-features = false }
-powdr-number = { git = "https://github.com/powdr-labs/powdr.git", rev = "c2ef9e4", default-features = false }
-powdr-constraint-solver = { git = "https://github.com/powdr-labs/powdr.git", rev = "c2ef9e4", default-features = false }
+powdr-openvm = { git = "https://github.com/powdr-labs/powdr.git", rev = "bbfc865", default-features = false }
+powdr-riscv-elf = { git = "https://github.com/powdr-labs/powdr.git", rev = "bbfc865", default-features = false }
+powdr-number = { git = "https://github.com/powdr-labs/powdr.git", rev = "bbfc865", default-features = false }
+powdr-autoprecompiles = { git = "https://github.com/powdr-labs/powdr.git", rev = "bbfc865", default-features = false }
```

Comment:
I'd say we can keep this commit now for initial testing, and then test the latest commit separately before merging that and doing CI.